OnlineDB/EcalCondDB/sql/trigger_add_doubleweights.sql

```diff
@@ -0,0 +1,166 @@
+
+insert into FE_CONFIG_WEIGHT2_INFO (wei2_conf_id, number_of_groups, TAG) values (1,0, 'NoOddWeights' ) ;
+
+
+create table FE_WEIGHT2_PER_GROUP_DAT(
+ wei2_conf_id number not null,
+ group_id number(10) not null,
+ W0 NUMBER,
+ W1 NUMBER,
+ W2 NUMBER,
+ W3 NUMBER,
+ W4 NUMBER,
+ W5 NUMBER
+ );
+
+ALTER TABLE FE_WEIGHT2_PER_GROUP_DAT ADD CONSTRAINT FE_WEIGHT2_PER_GROUP_pk PRIMARY KEY (wei2_conf_id , group_id);
+ALTER TABLE FE_WEIGHT2_PER_GROUP_DAT ADD CONSTRAINT FE_WEIGHT2_PER_GROUP_fk foreign KEY (wei2_conf_id) REFERENCES FE_CONFIG_WEIGHT2_INFO (wei2_conf_id);
+
+insert into FE_CONFIG_WEIGHT2_PER_GROUP (1,0,0,0,0,0,0,0);
+insert into FE_CONFIG_WEIGHT2_PER_GROUP (1,1,0,0,0,0,0,0);
+
+
+CREATE TABLE FE_CONFIG_WEIGHT2_DAT (
+ wei2_conf_id NUMBER NOT NULL,
+ logic_id NUMBER(10) not null, -- ( of the strip)
+ group_id number(10) not null);
+
+ALTER TABLE FE_CONFIG_WEIGHT2_DAT ADD CONSTRAINT FE_CONFIG_WEIGHT2_fk  FOREIGN KEY (wei2_conf_id) REFERENCES FE_CONFIG_WEIGHT2_INFO (wei2_conf_id);
+ALTER TABLE FE_CONFIG_WEIGHT2_DAT ADD CONSTRAINT FE_CONFIG_WEIGHT2_fk2  FOREIGN KEY (wei2_conf_id, group_id) 
+REFERENCES FE_WEIGHT2_PER_GROUP_DAT (wei2_conf_id, group_id);
+
+insert into FE_CONFIG_WEIGHT2_DAT (wei2_conf_id,logic_id,group_id) (select 1,logic_id,group_id from FE_CONFIG_WEIGHT_DAT where wei_conf_id=480);
+
+
+create table FE_WEIGHT2_MODE_DAT(
+ wei2_conf_id number not null,
+ EnableEBOddFilter number DEFAULT 0 NOT NULL,
+ EnableEEOddFilter number DEFAULT 0 NOT NULL,
+ EnableEBOddPeakFinder number DEFAULT 0 NOT NULL,
+ EnableEEOddPeakFinder number DEFAULT 0 NOT NULL,
+ DisableEBEvenPeakFinder number DEFAULT 0 NOT NULL,
+ FenixEBStripOutput number DEFAULT 0 NOT NULL,
+ FenixEEStripOutput number DEFAULT 0 NOT NULL,
+ FenixEBStripInfobit2 number DEFAULT 0 NOT NULL,
+ FenixEEStripInfobit2 number DEFAULT 0 NOT NULL,
+ EBFenixTcpOutput number DEFAULT 0 NOT NULL,
+ EBFenixTcpInfobit1 number DEFAULT 0 NOT NULL,
+ FenixPar12  number DEFAULT 0 NOT NULL,
+ FenixPar13  number DEFAULT 0 NOT NULL,
+ FenixPar14  number DEFAULT 0 NOT NULL,
+ FenixPar15  number DEFAULT 0 NOT NULL
+ );
+
+ALTER TABLE FE_WEIGHT2_MODE_DAT ADD CONSTRAINT FE_WEIGHT2_MODE_pk PRIMARY KEY (wei2_conf_id);
+ALTER TABLE FE_WEIGHT2_MODE_DAT ADD CONSTRAINT FE_WEIGHT2_MODE_fk foreign KEY (wei2_conf_id) REFERENCES FE_CONFIG_WEIGHT2_INFO (wei2_conf_id);
+
+INSERT into FE_WEIGHT2_MODE_DAT(wei2_conf_id) values (1); 
+
+
+
+CREATE SEQUENCE FE_CONFIG_WEIGHT2GROUP_SQ INCREMENT BY 1 START WITH 1;
+select FE_CONFIG_WEIGHT2GROUP_SQ.NextVal from DUAL; 
+
+-- this selects the first number that we have already used
+
+/*
+ *  the id is auto-incremented at each time you insert in the table 
+ *  no need to bother about inserting conf_id
+ */
+
+CREATE trigger FE_CONFIG_WEI2_TRG
+before insert on FE_CONFIG_WEIGHT2_INFO
+for each row
+begin
+select FE_CONFIG_WEIGHT2GROUP_SQ.NextVal into :new.wei2_conf_id from dual;
+end;
+/
+
+
+
+
+
```

Comment:
@valsdav I guess this script is used only in DB deployment... does it make sense to save it in the CMSSW repo? Are you planning to use a CMSSW release in for DB operation?

Reply:
I have added that script to the repo for documentation reasons. It contains the changes applied to the DB for the new double weights features.  It won't be used directly for DB operations (which have been already done).